### ISSUE TITLE
Support File Service REST API

### ISSUE THREAD

**Issue by crwilcox** [Azure/azure-storage-python#12]
### Description:
http://msdn.microsoft.com/en-us/library/azure/dn167006.aspx

**Comment by jehine-MSFT** [comment]
We have added support for all File REST APIs in the '2015-04-05' Azure Storage REST version. These additions will be rolled out in our next major release.

**Comment by schaefi** [comment]
Thanks for letting us know about it. Just a quick question; Am I correct that the changes in this release will stay compatible with the current file share methods:

```
from azure.storage.file import FileService
...
self.files = FileService(
    self.account_name, self.account_key
)
self.files.list_shares...
self.files.create_share...
self.files.delete_share...
```

Thanks

**Comment by jehine-MSFT** [comment]
There will be a lot of breaking changes across our library for our next release. In terms of the File service, there will be new APIs added for the latest REST version and there will be some changes to the interface to maximize ease of use. The breaking changes log will detail all the changes we've maid so you can update your code accordingly.

Thanks!

**Comment by jehine-MSFT** [comment]
Hello @schaefi & @dominoFire,

We have released the 0.30 version (#132) which addressed this feature request. If you have issues with the new version in regards to this issue, please reopen it and continue the discussion.

Thanks!
